src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -672,12 +676,22 @@ private void SetDocumentFlags(DocumentItem item, BuildVersionType buildType)
             }
         }
 
-        private bool SkipDocument(DocumentItem item, BuildVersionType buildType)
+        private bool SkipDocument(DocumentItem item, BuildVersionType buildType, IDocumentActions actions, SmugglerProgressBase.CountsWithLastEtag resultTombstones, out bool isLegacyDelete)
```

Comment:
I do not like that this method is handling delete markers inside. Handle them properly outside. Maybe in TryHandleLegacyDocuments() or something similar.

Reply:
Done